packages/errors/src/errors.ts

```diff
@@ -1350,19 +1350,29 @@ export const AllCypressErrors = {
     return errTemplate`\
     The ${fmt.highlight(`justInTimeCompile`)} configuration is only supported for Component Testing.`
   },
+  // TODO: link to docs on the new injectDocumentDomain config option
```

Comment:
@cacieprins Is this wanted to be done before 14 releases? If so, I would add an on link and we can route it to the https://docs.cypress.io/app/references/error-messages where the specific error explanation would be in the 14 docs branch.

Reply:
Yep, that work is in progress